src/views/import-pocket/importPocket.jsx

```diff
@@ -152,7 +153,7 @@ export default function ImportPocket() {
         return false;
       }
 
-      PocketService.saveUserInCache(
+      updateUser(
```

Comment:
I just realized that the code above this might be leading to a lot of issues as well. If we're importing from the private key, we should export the private key and not the PPK, as for exporting the PPK we need the passphrase used for creating it in the first place. I'm not sure if importing with the PK and then exporting the PPK "changes" the passphrase, but this is just really confusing behavior.

As this is old behavior and worked "Fine" for a while, we'll see how it goes, but, I'm suspicious about this part of the codebase now.